xbmc/pvr/PVRManager.cpp

```diff
@@ -97,6 +97,7 @@ CPVRManager::CPVRManager(void) :
     m_timers(NULL),
     m_addons(NULL),
     m_guiInfo(NULL),
+    m_parentalTimer(nullptr),
```

Comment:
while it's extremely unclear to me why this is a pointer in the first place, isn't it better to move to unique_ptr while at it ?